nl/guides/students/exercises/README.md

```diff
@@ -1,8 +1,12 @@
 ---
-title: Oefeningen oplossen
+title: "[nl] Oefeningen oplossen"
```

Comment:
It seems like you put the English verion here and the Dutch version in the `en` directory.